package.json

```diff
@@ -2,7 +2,10 @@
   "name": "resin-raven",
   "version": "1.0.0-alpha.1",
   "description": "A wrapper around Sentry and Mixpanel that works in the browser, Node and Electron",
-  "main": "index.js",
+
+  "main": "src/services/external/node.js",
```

Comment:
I don't quite get this. These files don't export anything. They require the top-level file (index) and inject some of the dependencies, but don't return anything to the outer world.

So when I do `require('resin-raven')` I would get an empty exports object.

Am I wrong?

I would imagine them looking like

```
const resinRavenFactory = require('./common')

module.exports = resinRavenFactory({
  SentryLib: require('raven-js/dist/raven'),
  MixpanelLib: require('mixpanel-browser')
})
```

Also reaching out to the higher-level modules is usually a code smell, normally the parent should assmeble its children.